packages/text-document-utils/src/index.ts

```diff
@@ -0,0 +1,52 @@
+export {
+    TextDocument,
+    Position,
+    Range,
+    DocumentUri,
+    TextDocumentContentChangeEvent,
+    TextEdit
+} from 'vscode-languageserver-textdocument';
+
+export {
+    Location,
+    Range as RangeType,
+    Position as PositionType,
+    Diagnostic,
+    DiagnosticSeverity,
+    TextEdit as TextEditType,
```

Comment:
Why are we exporting two versions of the same type here?

Reply:
updated exports